packages/graalvm/api/graalvm.api

```diff
@@ -1075,3 +1092,78 @@ public final class elide/runtime/intrinsics/js/err/ValueError$Factory : elide/ru
 	public fun create (Ljava/lang/Throwable;)Lelide/runtime/intrinsics/js/err/ValueError;
 }
 
+public abstract interface class elide/runtime/intrinsics/js/http/HttpAPI {
+	public abstract fun createServer (Ljava/lang/Object;Lorg/graalvm/polyglot/Value;)Lelide/runtime/intrinsics/js/http/Server;
+	public static synthetic fun createServer$default (Lelide/runtime/intrinsics/js/http/HttpAPI;Ljava/lang/Object;Lorg/graalvm/polyglot/Value;ILjava/lang/Object;)Lelide/runtime/intrinsics/js/http/Server;
+}
```

Comment:
should `internal`ize as many of these classes as possible, lest we provide surface which users may consume